about.html

```diff
@@ -28,7 +28,7 @@
                 <div class="github-icon">
                     <a href="https://github.com/arpitghura/typing-test" class="nav-item-link"><i class='bx bx-sm bxl-github'></i></a>
                 </div>
-                <button onclick="toggleLightDarkMode()" class="dark-btn">Dark Mode</button>     
+                <i id="icon1" onlick="change('icon1')" onclick="toggleLightDarkMode()" class="bx bx-moon dark-btn" style="font-size: x-large;"></i>     
```

Comment:
What is `onlick="change('icon1')"` ? I am not sure that onlick is an event listener, do you want to use `onclick`?

Reply:
yea  i guess i made a typo

Reply:
@arpitghura let me fix that